src/goFormat.ts

```diff
@@ -34,7 +34,7 @@ export class GoDocumentFormattingEditProvider implements vscode.DocumentFormatti
 			formatFlags.push('-style=indent=' + options.tabSize);
 		}
 
-		return this.runFormatter(formatTool, formatFlags, document).then(edits => edits, err => {
+		return this.runFormatter(formatTool, formatFlags, document, token).then(edits => edits, err => {
```

Comment:
Have you found a scenario where this `token` was actually cancelled by vscode?

Reply:
In that case what is the need to have our own token?

Reply:
Bit confused. The token just signals that the result will be discarded. If we don't listen to it and kill the process ourselves wouldn't it keep going?